R/int_pctl.R

```diff
@@ -67,21 +69,27 @@
 #'   int_pctl(lm_res)
 #' }
 #' @export
-int_pctl.tune_results <- function(.data, metrics = NULL, eval_time = NULL,
-                                  times = 1001, parameters = NULL,
-                                  alpha = 0.05, allow_par = TRUE,
-                                  event_level = "first", ...) {
-
+int_pctl.tune_results <- function(
+  .data,
+  metrics = NULL,
+  eval_time = NULL,
+  times = 1001,
+  parameters = NULL,
+  alpha = 0.05,
+  allow_par = TRUE,
+  event_level = "first",
+  keep_replicates = FALSE,
+  ...
+) {
   rlang::check_dots_empty()
 
-  # performance speedup, enables tidymodels/yardstick#428
-  initialize_catalog(control_resamples(allow_par = TRUE))
```

Comment:
Might loop Emil in here to ensure we don't lose this speedup with yardstick

Reply:
We're a go for removing it